services/worker/worker.py

```
@@ -795,9 +858,187 @@ def _dispatch_pipeline_step(
+    workflow_run_id: str | None = None,
+    payload: dict | None = None,
+) -> dict:
+    now = datetime.now(timezone.utc)
+    details = dict(payload or {})
+
+    with Session(get_engine()) as session:
+        workflow_uuid: UUID | None = None
+        if workflow_run_id:
+            try:
+                workflow_uuid = UUID(workflow_run_id)
+            except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)